src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -840,9 +840,12 @@ def main(argv):
   if use_dataflow_tracing:
     dataflow_binary_path = os.path.join(
         dataflow_build_dir, os.path.relpath(fuzzer_path, build_directory))
-    arguments.append(
-        '%s%s' % (constants.COLLECT_DATA_FLOW_FLAG, dataflow_binary_path))
-    fuzzing_strategies.append(strategy.DATAFLOW_TRACING_STRATEGY.name)
+    if os.path.exists(dataflow_binary_path):
+      arguments.append(
+          '%s%s' % (constants.COLLECT_DATA_FLOW_FLAG, dataflow_binary_path))
+      fuzzing_strategies.append(strategy.DATAFLOW_TRACING_STRATEGY.name)
+    else:
+      logs.log_warn('Fuzz target is not found in dataflow build.')
```

Comment:
log_error as otherwise we will never see it on our dashboards.

also, can add ", skip strategy" in message.

Reply:
Done and done. However, I'm not sure whether we want to see it on the error dashboard. Since we don't synchronize build revisions, we'll likely be hitting this from time to time, and we'll be just ignoring that error then. IMO such errors make the dashboard hard to navigate / triage :(